source/module_psi/CMakeLists.txt

```diff
@@ -4,6 +4,17 @@ add_library(
     psi.cpp
 )
 
+add_library(
```

Comment:
Maybe add a new submodule in module_psi is better.

Reply:
Not sure, because hear from pw maintainers the psi is already in refactor plan, so it is also a temporary choice to place it here...